draft-cds-rats-intel-corim-profile.md

```diff
@@ -771,18 +778,29 @@ Alternatively, for Evidence, the TEE epoch timestamp may be encoded using `mkey`
 
 ### The tee-instance-id-type Measurement Extension {#sec-tee-instance-id-type}
 
-The `tee.instance-id` extension enables the Attester to report the (TBD:instance-id-description) Evidence value
-and the RVP to assert an exact-match Reference Value.
-
-The `$tee-instance-id-type` is an unsigned integer.
+The `tee.instance-id` extension enables the Attester to report the TEE (TD or enclave) instance identifier as an Evidence value and the RVP to assert an exact-match Reference Value.
 
-The `$tee-instance-type` is an exact match measurement.
+The `$tee-instance-id-type` is an unsigned integer or `bstr`.
 
 ~~~ cddl
 {::include cddl/tee-instance-id-type.cddl}
 ~~~
 
-Alternatively, the TEE instance ID may be encoded using `mkey` where `mkey` contains the non-negative `tee.instance-id` and `mval`.`raw-value` contains the `$tee-instance-id-type` value.
+Alternatively, the TEE instance ID may be encoded using `mkey` where `mkey` contains the non-negative `tee.instance-id` and `mval`.`raw-value` contains the `$tee-instance-id-type` value. If the `$tee-instance-id-type` is an unsigned integer, the integer values is converted to a single byte `bstr`.
+
+### The tee-platform-instance-id-type Measurement Extension {#sec-tee-platform-instance-id-type}
+
+Platform Instance ID is a globally unique identifier generated by the platform during Platform Establishment. This value remains consistent across trusted computing base (TCB) recoveries, but is regenerated during Platform Establishment due to desire to reset keys or to add and remove hardware. See (Section 3.7 {{-dcap}}).
+
+The `tee.platform-instance-id` extension enables the Attester to report the platform instance identifier as an Evidence value and the RVP to assert an exact-match Reference Value.
+
+The `$tee-platform-instance-id-type` is a `bstr`.
```

Comment:
ANy reason why it is not symmetrical to the other instance id in the system.

I would hope we use consistent identities in the system..?

Reply:
I mean tee-instance-id..?

Reply:
The tee instance ID combines a couple of identifiers that were typed differently but have the same basic semantics. Platform instance doesn't have that dynamic.